site/content/in-dev/unreleased/polaris-spark-client.md

```diff
@@ -0,0 +1,176 @@
+
+## Quick Start with Local Polaris service
+If you want to quickly try out the functionality with a local Polaris service, you can follow the instructions
+in `plugins/spark/v3.5/getting-started/README.md`.
+
+The getting-started will start two containers:
+1) The `polaris` service for running Apache Polaris using an in-memory metastore
+2) The `jupyter` service for running Jupyter notebook with PySpark (Spark 3.5.5 is used)
+
+The notebook `SparkPolaris.ipynb` provided under `plugins/spark/v3.5/getting-started/notebooks` provides examples 
+with basic commands, includes:
+1) Connect to Polaris using Python client to create a Catalog and Roles
+2) Start Spark session using the Polaris Spark client
+3) Using Spark to perform table operations for both Delta and Iceberg
+
+## Start Spark against a deployed Polaris service
+Before starting, make sure the service deployed is up-to-date, and that Spark 3.5 with at least version 3.5.3 is installed.
+Spark 3.5.5 is recommended, and you can follow the instructions below to get a Spark 3.5.5 distribution.
+```shell
+cd ~
+wget https://archive.apache.org/dist/spark/spark-3.5.5/spark-3.5.5-bin-hadoop3.tgz 
+mkdir spark-3.5
+tar xzvf spark-3.5.5-bin-hadoop3.tgz  -C spark-3.5 --strip-components=1
+cd spark-3.5
+```
+
+### Connecting with Spark using the Polaris Spark client
+The following CLI command can be used to start the spark with connection to the deployed Polaris service using
+a released Polaris Spark client.
+
+```shell
+bin/spark-shell \
+--packages <polaris-spark-client-package>,org.apache.hadoop:hadoop-aws:3.4.0,io.delta:delta-spark_2.12:3.3.1 \
+--conf spark.sql.extensions=org.apache.iceberg.spark.extensions.IcebergSparkSessionExtensions,io.delta.sql.DeltaSparkSessionExtension \
+--conf spark.sql.catalog.spark_catalog=org.apache.spark.sql.delta.catalog.DeltaCatalog \
+--conf spark.sql.catalog.<spark-catalog-name>.warehouse=<polaris-catalog-name> \
+--conf spark.sql.catalog.<spark-catalog-name>.header.X-Iceberg-Access-Delegation=vended-credentials \
+--conf spark.sql.catalog.<spark-catalog-name>=org.apache.polaris.spark.SparkCatalog \
+--conf spark.sql.catalog.<spark-catalog-name>.uri=<polaris-service-uri> \
+--conf spark.sql.catalog.<spark-catalog-name>.credential='<client-id>:<client-secret>' \
+--conf spark.sql.catalog.<spark-catalog-name>.scope='PRINCIPAL_ROLE:ALL' \
+--conf spark.sql.catalog.<spark-catalog-name>.token-refresh-enabled=true
+```
+Assume the released Polaris Spark client you want to use is `org.apache.polaris:polaris-iceberg-1.8.1-spark-runtime-3.5_2.12:1.0.0`,
+replace the `polaris-spark-client-package` field with the release.
+
+The `spark-catalog-name` is the catalog name you will use with Spark, and `polaris-catalog-name` is the catalog name used 
+by Polaris service, for simplicity, you can use the same name. 
+
+Replace the `polaris-service-uri`, `client-id` and `client-secret` accordingly, you can refer to
```

Comment:
`polaris-service-uri` is not described in "Using Polaris" - you should state what the correct value should be here based on a standard local Spark deployment.